services/worker/src/worker/job_runners/config/parquet_and_info.py

```diff
@@ -1241,11 +1241,7 @@ def compute_config_parquet_and_info_response(
         raise EmptyDatasetError(f"{dataset=} is empty.", cause=err) from err
     except ValueError as err:
         if "trust_remote_code" in str(err):
-            raise DatasetWithScriptNotSupportedError(
-                "The dataset viewer doesn't support this dataset because it runs "
-                "arbitrary python code. Please open a discussion in the discussion tab "
-                "if you think this is an error and tag @lhoestq and @severo."
-            ) from err
+            raise DatasetWithScriptNotSupportedError from err
```

Comment:
btw, we should have passed the cause error as a parameter:

```suggestion
            raise DatasetWithScriptNotSupportedError(cause=err) from err
```

It's unrelated to the subject of this PR though, and we surely could be smarted and get the value automatically when we use `from err`

Reply:
Yes, we can get the value automatically from the `__cause__` attribute of the raised Exception with a `from` clause.

Reply:
https://github.com/huggingface/dataset-viewer/issues/2753